src/mcts/search.h

```diff
@@ -303,7 +303,7 @@ class SearchWorker {
   };
 
   NodeToProcess PickNodeToExtend(int collision_limit);
-  void ExtendNode(Node* node);
+  void ExtendNode(Node* node, int depth = 0);
```

Comment:
The default value isn't necessary.

Reply:
Originally I didn't realize that there is only 1 place where `ExtendNode()` is called, which I modified.